### PR TITLE
Backup: make steps procedural, add related links

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Backup: make steps procedural, add related links
- Style change: use numbered steps to make steps procedural
- Add related topics
- Add glossary term refs
- Remove manual configuration section

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/backup.markdown [hunk 1]
```diff
@@ -72,15 +56,43 @@ automation:
...
 
-If you use Home Assistant Operating System or Home Assistant Supervised, [the restore functionality is already built-in](/common-tasks/os/#restoring-a-backup).
+The steps on recovering from a backup depend on the installation type and use case. Follow one of these steps:
 
-</div>
+1. If you use {% term "Home Assistant Operating System" %} or {% term "Home Assistant Supervised" %}:
+   - The restore functionality is already built-in.
+   - Follow the steps described in [Restoring a backup](/common-tasks/os/#restoring-a-backup).
+2. If you use a {% term "Home Assistant Core" %} or {% term "Home Assistant Container" %} installation:
+   - A Home Assistant backup is just a tar file of the `/config` directory, plus some metadata.
```
Comment (ludeeus) [inline source/_integrations/backup.markdown]:
```suggestion
   - A Home Assistant backup is just a tar file of the [configuration directory](/docs/configuration/#editing-configurationyaml), plus some metadata.
```
`/config` only exists in the containers, not for core so we should link to the explanation here.

File: source/_integrations/backup.markdown [hunk 2]
```diff
@@ -72,15 +56,43 @@ automation:
...
-<div class="note">
+Backups created via the **Backup** integration are located in your `/config/backups` directory.
+The {% term "Home Assistant Container" %} installation typically mounts this directory via `docker-compose.yml` or `docker run` to a directory of your choice.
 
-If you use Home Assistant Operating System or Home Assistant Supervised, [the restore functionality is already built-in](/common-tasks/os/#restoring-a-backup).
+The steps on recovering from a backup depend on the installation type and use case. Follow one of these steps:
 
-</div>
+1. If you use {% term "Home Assistant Operating System" %} or {% term "Home Assistant Supervised" %}:
+   - The restore functionality is already built-in.
```
Comment (ludeeus) [inline source/_integrations/backup.markdown]:
```suggestion
```
This is implied with the next step.

File: source/_integrations/backup.markdown [hunk 3]
```diff
@@ -72,15 +56,43 @@ automation:
...
+1. In your [configuration file](/docs/configuration/), re-add the following entry:
+
+    ```yaml
+    # Example configuration.yaml entry
+    backup:
+    ```
+
+2. Go to {% my system_dashboard title="**Settings**" %} and in the top right corner, select the three dots menu and select **Restart** > **Restart Home Assistant**.
+   - **Result**: When it has started up again, you will find a new **Backup** entry under ({% my backup title="**Settings** > **System** > **Backups**" %}**).
+   - The backup files are stored in a new **backups** subdirectory in the root of your configuration directory, `/config/backups/`.
```
Comment (ludeeus) [inline source/_integrations/backup.markdown]:
```suggestion
```

While you _can_ add this yo your configuration, doing so does nothing as this integration is always loaded regardless.
https://github.com/home-assistant/core/blob/a213de3db5524b065d56f9ba7a9d7d187ec6176b/homeassistant/bootstrap.py#L171

File: source/_integrations/backup.markdown [hunk 4]
```diff
@@ -72,15 +56,43 @@ automation:
 
 ## Restoring a backup
 
-<div class="note">
+Backups created via the **Backup** integration are located in your `/config/backups` directory.
```
Comment (ludeeus) [inline source/_integrations/backup.markdown]:
```suggestion
Backups created via the **Backup** integration are located in your a subdirectory named `backups` which are located in your [configuration directory](/docs/configuration/#editing-configurationyaml).
```

**Review by ludeeus — CHANGES_REQUESTED** [review]

**Review by ludeeus — APPROVED** [review]
LGTM 👍 
But still needs @home-assistant/core to look as well.

File: source/_integrations/backup.markdown [hunk 5]
```diff
@@ -72,15 +56,28 @@ automation:
 
 ## Restoring a backup
 
-<div class="note">
+Backups created via the **Backup** integration are located in your a subdirectory named `backups` which are located in your [configuration directory](/docs/configuration/#editing-configurationyaml).
```
Comment (MartinHjelmare) [inline source/_integrations/backup.markdown:59]: Remove "your" in "located in your a subdirectory".

Comment (c0ffeeca7) [reply source/_integrations/backup.markdown:59]: Thank you! https://github.com/home-assistant/home-assistant.io/pull/32318

**Pull request merged**